### PR TITLE
Update rules and create rules json

### PR DESCRIPTION
**Pull request by mduleone**
This adds a `print` npm script, which runs the `/print.js` file and dumps the resultant rules in to `/linters/eslintrc.json`

Because of that, we can ignore the `eslintrc.json` file as a part of this PR, as it is auto-generated by the rules.

### CODE REVIEW

File: linters/eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,1712 @@
...
+    "array-bracket-newline": [
+      "error",
+      {
+        "multiline": true
+      }
+    ],
+    "array-element-newline": [
+      "error",
+      {
+        "multiline": true,
```
Comment (Korzun) [inline linters/eslintrc.json]: This setup appears to require arrays to be on one line, unless a child is multi-line. I think this is a whoopsie-daisy. The following seems to be a more reasonable setting:
```
"array-element-newline": [
  "error",
  "always"
],
```

Comment (mduleone) [reply linters/eslintrc.json]: :+1: good catch! Updated in aa1218cea1c21d3b1bfd23d7b9401934a1473163

File: packages/eslint-config-airbnb-base/rules/best-practices.js [hunk 2]
```diff
@@ -169,19 +170,7 @@ module.exports = {
...
-        'e', // for e.returnvalue
-        'ctx', // for Koa routing
-        'req', // for Express requests
-        'request', // for Express requests
-        'res', // for Express responses
-        'response', // for Express responses
-        '$scope', // for Angular 1 scopes
-      ]
-    }],
+    'no-param-reassign': 'off',
```
Comment (danielchangNYC) [inline packages/eslint-config-airbnb-base/rules/best-practices.js]: why turn this off?

File: packages/eslint-config-airbnb-base/rules/style.js [hunk 3]
```diff
@@ -93,8 +91,7 @@ module.exports = {
 
     // enforces use of function declarations or expressions
     // https://eslint.org/docs/rules/func-style
-    // TODO: enable
-    'func-style': ['off', 'expression'],
+    'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
```
Comment (danielchangNYC) [inline packages/eslint-config-airbnb-base/rules/style.js]: why declaration and not expression?

**Review by dannyc5 — APPROVED** [review]

**Review by Korzun — APPROVED** [review]
👏 👏 👏

**Pull request merged**